src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1305,12 +1342,15 @@ private void UpdateEditor(float delta)
     /// </summary>
     private void RenderHighlightedOrganelle(int q, int r, int rotation)
     {
-        if (ActiveActionName == null)
+        if (movingOrganelle == null && ActiveActionName == null)
             return;
 
         // TODO: this should be changed into a function parameter
         var toBePlacedOrganelle = SimulationParameters.Instance.GetOrganelleType(
-            ActiveActionName);
+            movingOrganelle != null
+            ? movingOrganelle.Definition.InternalName
```

Comment:
`movingOrganelle.Definition` is already the organelle type, no need to get it's name and then lookup the object again, causes unnecessary dictionary lookup.